Bottom: dc2232cbe4b5f59458b29964d9cf41ff87ed4e3b
Top:    67ca83e6669a7335c983e1a20288ddc424957f60
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 17:08:13 +0100

Refresh of report-c-don-t-leave-actions-i

---

diff --git a/src/report.c b/src/report.c
index 07d7d94..0467e80 100644
--- a/src/report.c
+++ b/src/report.c
@@ -803,16 +803,16 @@ ReportNetLengthByName (char *tofind, int x, int y)
 	      regerror (result, &elt_pattern, errorstring, 128);
 	      Message (_("regexp error: %s\n"), errorstring);
 	      regfree (&elt_pattern);
-              retval = 1;
-              goto out;
+	      retval = 1;
+	      goto out;
 	    }
 #endif
 #if defined(HAVE_RE_COMP)
 	  if ((elt_pattern = re_comp (tofind)) != NULL)
 	    {
 	      Message (_("re_comp error: %s\n"), elt_pattern);
-              retval = 1;
-              goto out:
+	      retval = 1;
+	      goto out:
 	    }
 #endif
 	}
